modules/setup.rb

```diff
@@ -9,67 +9,8 @@ class << self
-    puts 'You don\'t have a valid config file!'
-    puts 'If you want to create a config file manually, please stop this program with Ctrl+C ' \
-      'and follow the instructions in config/README.md.'
-    puts 'Press any key to continue!'
-    STDIN.getch
-
-    # Initialise a fresh set of configuration options.
-    newconfig = YAML.load_file('config/config.sample.yml')
-    puts '-=-=-=-=-=-=-=-=-=-=-=-=-=-=-'
-    puts 'Okay! First of all you\'ll need to create a bot account!'
-    puts 'To do that, go here -> https://discordapp.com/developers/applications/me <-'
-    puts 'And login to your Discord account! Then follow the instructions to make a new app'
-    puts 'After you have your app, select the "Bot" tab and click the blue button ' \
-      'labelled "Add Bot"'
-    puts 'Then click "click to reveal" next to \'Token:\' and copy the long string inside!'
-    print 'Please paste that string here: '
-    token = gets.chomp
-
-    puts '-=-=-=-=-=-=-=-=-=-=-=-=-=-=-'
-    puts 'Awesome! Now you\'ll need to head back to the "General information" tab' \
-      'and fetch the Client ID, which should be a long number.'
-    print 'And paste that here: '
-    appid = gets.chomp
-
-    puts '-=-=-=-=-=-=-=-=-=-=-=-=-=-=-'
-    puts 'Wew, that\'s the hard part out of the way. ' \
-      'Now you get to decide some needed options for your bot!'
-    puts 'Your bot will need at least one prefix to be used. For example: ' \
-      'if you use the prefix s!, your help command will be \'s!help\'.'
-    puts 'If your prefix is a word, please add a space after it! ' \
-      '(For example enter \'tomoe\' for \'tomoe ping\' instead of \'tomoeping\''
-    print 'Enter your first prefix here: '
-    prefix = gets.chomp
-
-    puts "Nice prefix! Your bots help command will be: '#{prefix}help'"
-
-    puts '=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-'
-    puts 'That\'s enough information to get the bot launching, but there\'s one more thing we need.'
-    puts 'If you haven\'t already, go to User Settings -> Appearance, and enable Developer Mode.'
-    puts 'Then, right click your own name and select Copy ID.'
-    print 'Paste that ID here: '
-    id = gets.chomp
-    puts 'Nice! This will make you the owner of the bot and allow you to control things ' \
-      'and add more owners through commands.'
-
-    puts "\nPlease note that there's more to verything configured above!"
-    puts 'Check config/config.sample.yml and config.yml to fine-tune the options.'
-
-    puts "\n We're now collating the information and generating a configuration file..."
-    newconfig['token'] = token
-    newconfig['client_id'] = appid
-    newconfig['prefixes'][0] = prefix
-    newconfig['master_owner'] = id
-
-    File.write('config/config.yml', newconfig.to_yaml)
-
-    puts 'All done! The bot will launch in 1 second..'
-    # Delay so the user knows what's happening.
-    sleep(1)
-
-    # Let's go!
+    puts 'No valid config file found. Please copy config/config.sample.yml
```

Comment:
We should probably exit here, so that there aren't odd errors about missing/nil so forth afterwards.

Reply:
I had considered this, the resulting traceback says, among other things, `No such file or directory @ rb_sysopen - config/config.yml` which I thought was an appropriate reason to exit, can change to cleanly exit if you disagree though.

Reply:
```
erisa@Tuturu /m/c/U/E/O/W/B/Yuuki-Bot (master)> bash run_linux.sh
No valid config file found. Please copy config/config.sample.yml to config/config.yml and edit it, or run config.rb.
Traceback (most recent call last):
        7: from run.rb:3:in `<main>'
        6: from run.rb:30:in `<module:YuukiBot>'
        5: from run.rb:30:in `require_relative'
        4: from /mnt/c/Users/Erisa/OneDrive/Work/Bot/Yuuki-Bot/modules/setup.rb:3:in `<top (required)>'
        3: from /mnt/c/Users/Erisa/OneDrive/Work/Bot/Yuuki-Bot/modules/setup.rb:15:in `<module:YuukiBot>'
        2: from /usr/lib/ruby/2.7.0/psych.rb:577:in `load_file'
        1: from /usr/lib/ruby/2.7.0/psych.rb:577:in `open'
/usr/lib/ruby/2.7.0/psych.rb:577:in `initialize': No such file or directory @ rb_sysopen - config/config.yml (Errno::ENOENT)```

Reply:
Ah alright, sounds good - I didn't actually give this a run. After more thorough testing, looks good to me.